### DISCUSSION TITLE
[1.16.5] Not Enough Chocolate

### DISCUSSION THREAD

**Discussion by MoogieOuttaMyDepth** [vectorwing/FarmersDelight#441] [Ideas]
Good evening. I've spotted a glaring bug in what is otherwise a fantastic mod: There is only one chocolate recipe!

As a lifelong chocolate connoisseur, I was both shocked and horrified to discover there is only one chocolate recipe in the entire mod. After calming myself down, I realized this is _obviously_ an error. Who would deliberately only include 1 recipe for the most important and delicious kind of food to have ever been invented?? And for that recipe to be chocolate pie, as opposed to something more classic, like your basic, run-of-the-mill (but still mouthwateringly delicious) chocolate bar? That would be insanity!

Undoubtedly, you _did_ intend to include more varieties of chocolate treats, such as: milk/dark/white chocolate bars, chocolate biscuits, chocolate truffles, chocolate mini-rolls, chocolate brownies, chocolate chip cookies, chocolate cakes, black forest gateau, chocolate cupcakes, chocolate mousse, chocolate milkshake, chocolate macarons, chocolate crepes, chocolate ice cream, and chocolate soufflés.

But somehow, they just don't show up in the current version. How weird!

I hope this issue can be addressed in future versions, as it is the only flaw I can find in the entirety of this mod. Even the addition of 1-2 more chocolate recipes would go a long way towards improving the situation. Until then, my dream of owning my very own Minecraft Chocolatier shop will have to patiently wait.

P.S. I hope you see this for the tongue-in-cheek suggestion that it is meant to be, and not some serious demand from a deranged chocolate lunatic. I _am_ a deranged chocolate lunatic, but I'm not demanding anything of you. Please keep up the wonderful work, Vectorwing. :)

**Comment by thelegitdolt** [comment]
you can try neapolitan it has chocolate and it's actually good

**Comment by vectorwing** [comment]
I can sense your choco-enthusiasm in Ultra 4K VR from here. 😆

I moved this one over to discussions, as it's more elaborate on being a suggestion. Yeah, perhaps I haven't explored chocolate too much yet, and I'm not sure it will go as deep as your suggestions; FD tries to keep things simple and low-quantity.

That said, I do plan to shoehorn chocolate in at least a few desserts in the future; just gotta plan it properly first. 😉
